tests/meshService.test.js

```diff
@@ -0,0 +1,146 @@
+import fs from "fs";
+import csvParser from "csv-parser";
+import path from "node:path";
+import dotenv from "dotenv";
+import payload from "../src/model/payload.js";
+import loader from "../src/service/loader.js"
+import senderService from "../src/service/senderService.js";
+import receiverService from "../src/service/receiverService.js"
+import meshService from "../src/service/meshService.js";
+
+// Empty input and output dirs
+async function emptyDirs() {
+  const inputDir = "./input";
+  const outputDir = "./output";
+
+  fs.readdir(inputDir, (err, files) => {
+    if (err) throw err;
```

Comment:
duplication of code with below function. Best to create a function that takes directory as a argument. Sonarqube will complain otherwise

Reply:
Ok will do

Reply:
Is this considered duplicate code by sonarqube?